HousingSearchListener.Tests/V1/UseCase/UpdateProcessUseCaseTests.cs

```diff
@@ -3,6 +3,7 @@
 using Hackney.Core.Sns;
 using Hackney.Shared.HousingSearch.Domain.Process;
 using Hackney.Shared.HousingSearch.Gateways.Models.Processes;
+using Hackney.Shared.Processes.Sns;
```

Comment:
Not sure why `ProcessMessageAsyncTestIndexProcessSuccess` passes, but I think it's worth verifying that `esProcess` should have the state name & startedAt from `_message.EventData.NewData` (this would be done by updating `VerifyProcessIndexed`)

Reply:
`ProcessMessageAsyncTestIndexProcessSuccess` passes, because of equivalent of those:
![image](https://user-images.githubusercontent.com/10539291/197174400-3e0c9df8-f0c5-4913-8f4d-93c6c0a90e84.png)
`VerifyProcessIndexed` updated and will be added to the PR

Reply:
Right, but it's not supposed to be equivalent as the processStartedAt and stateStartedAt variables should be changed

Reply:
here we set `newState` with the relevant data and on line 43 we set the `StateStartdAt` to be equal to `esProcess.StartedAt` and they should be equal or I am missing something?
![image](https://user-images.githubusercontent.com/10539291/197211274-19aa6415-0b06-4072-8cdf-99e6f665dbdf.png)

Reply:
Yes, looks good now